tests/h/services/document_test.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import pytest
+
+from h.services.document import DocumentService
+
+
+class TestFetchByGroupid(object):
+    def test_it_returns_documents_annotated_within_group(
+        self, svc, groups, annotations, matchers
+    ):
+        docs = svc.fetch_by_groupid(groupid=groups["target_group"].pubid)
+
+        assert docs == matchers.UnorderedList([anno.document for anno in annotations])
```

Comment:
I wonder if tests could access `annotations` as `target_group.annotations` instead of using the fixture directly? Might make it clearer that these are the annotations belonging to `target_group`. Similarly `other_annotations` as `other_group.annotations`.